src/main/kotlin/org/rust/ide/statistics/RsCodeVisionUsageCollector.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.statistics
+
+import com.intellij.internal.statistic.eventLog.EventLogGroup
+import com.intellij.internal.statistic.eventLog.events.EventFields
+import com.intellij.internal.statistic.eventLog.events.StringEventField
+import com.intellij.internal.statistic.service.fus.collectors.CounterUsagesCollector
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.RsMacroDefinitionBase
+
+@Suppress("UnstableApiUsage")
+class RsCodeVisionUsageCollector : CounterUsagesCollector() {
```

Comment:
You didn't register this extension, so it won't be possible to generate rules for this collector for statistics server